helm/istio/charts/tracing/templates/jaeger-all-in-one.yaml

```diff
@@ -0,0 +1,74 @@
+{{ if and (eq .Values.provider "jaeger") (eq .Values.jaeger.template "all-in-one") }}
+apiVersion: jaegertracing.io/v1
+kind: "Jaeger"
+metadata:
+  name: "jaeger"
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ template "tracing.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  strategy: allInOne
+
+  allInOne:
+    image: {{ .Values.jaeger.hub }}/all-in-one:{{ .Values.jaeger.tag }}
+    options:
+      log-level: debug
+      query:
+        base-path: /
+    annotations:
+      {{- range $key, $value := .Values.jaeger.annotations }}
+        {{ $key }}: {{ $value | quote }}
+      {{- end }}
+
+  agent:
+    strategy: {{ .Values.jaeger.agentStrategy }}
+    image: {{ .Values.jaeger.hub }}/jaeger-agent:{{ .Values.jaeger.tag }}
+    serviceAccount: jaeger-agent
+    options:
+      collector:
+        host-port: jaeger-collector:14267
+
+  ui:
+    options:
+      dependencies:
+        menuEnabled: false
+      gaTrackingID: UA-000000-2
```

Comment:
Is this a Google analytics tracking ID? Do we have to worry about this?

Reply:
@objectiser, should we remove the gaTrackingID?

Reply:
Yes it can be removed - I think it was just carried over from an example.